core/pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "covidflow"
+version = "0.1.0"
+description = ""
+authors = [""]
+
+[tool.poetry.dependencies]
+python = "^3.6"
+rasa-sdk = "^1.9.0"
```

Comment:
We should not be needing rasa-sdk here. We might eventually need rasa 1.9.4 tho.